edx-frontend-build-tools/config/webpack.dev.config.js

```diff
@@ -161,7 +126,7 @@ module.exports = Merge.smart(commonConfig, {
   // reloading.
   devServer: {
     host: '0.0.0.0',
-    port: 1995,
+    port: 8080,
```

Comment:
This part is interesting.  Wonder if we can get the port for each app from the .env file.

Reply:
Done and done!